bowler/tool.py

```diff
@@ -286,14 +287,25 @@ def process_hunks(self, filename: Filename, hunks: List[Hunk]) -> None:
                         raise ValueError("unknown response")
 
             if result == "y" or self.write:
-                patch = ("\n".join(hunk) + "\n").encode("utf-8")
-                args = ["patch", "-u", filename]
-                self.log_debug(f"running {args}")
-                try:
-                    sh.patch("-u", filename, _in=patch)  # type: ignore
-                except sh.ErrorReturnCode:
-                    log.exception("failed to apply patch hunk")
-                    return  # TODO: better response?
+                accepted_hunks += "\n".join(hunk[2:]) + "\n"
+
+        if accepted_hunks:
+            accepted_hunks = f"--- {filename}\n+++ {filename}\n" + accepted_hunks
```

Comment:
I believe it would be faster to just include `accepted_hunks` in the f-string, rather than rendering the f-string and then concatenating.